<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplace</class>
 <widget class="QWidget" name="FindAndReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quick find</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayoutReplace">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="replace">
         <property name="toolTip">
          <string>Replace</string>
         </property>
         <property name="statusTip">
          <string>Replace</string>
         </property>
         <property name="whatsThis">
          <string>Replace</string>
         </property>
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="replaceAll">
         <property name="toolTip">
          <string>Replace all</string>
         </property>
         <property name="statusTip">
          <string>Replace all</string>
         </property>
         <property name="whatsThis">
          <string>Replace all</string>
         </property>
         <property name="text">
          <string>Replace all</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacerReplace">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="FadClearLineEdit" name="replaceEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTextReplaceWith">
       <property name="text">
        <string>Replace with:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="FindFadLineEdit" name="findEdit">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTextFind">
       <property name="text">
        <string>Find:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayoutNextPrevious">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="previous">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip">
          <string>Previous</string>
         </property>
         <property name="statusTip">
          <string>Previous</string>
         </property>
         <property name="whatsThis">
          <string>Previous</string>
         </property>
         <property name="text">
          <string>Previous</string>
         </property>
         <property name="icon">
          <iconset resource="findandreplace.qrc">
           <normaloff>:/FindAndReplace/previous</normaloff>:/FindAndReplace/previous</iconset>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="next">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip">
          <string>Next</string>
         </property>
         <property name="statusTip">
          <string>Next</string>
         </property>
         <property name="whatsThis">
          <string>Next</string>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
         <property name="icon">
          <iconset resource="findandreplace.qrc">
           <normaloff>:/FindAndReplace/next</normaloff>:/FindAndReplace/next</iconset>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="close">
         <property name="toolTip">
          <string>Close</string>
         </property>
         <property name="statusTip">
          <string>Close</string>
         </property>
         <property name="whatsThis">
          <string>Close</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="icon">
          <iconset resource="../Note/Note.qrc">
           <normaloff>:/Note/remove</normaloff>:/Note/remove</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>bottomVerticalSpacer</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FadClearLineEdit</class>
   <extends>QLineEdit</extends>
   <header>FadLineEdit/fadclearlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>FindFadLineEdit</class>
   <extends>QLineEdit</extends>
   <header>FindAndReplace/findfadlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>replaceEdit</tabstop>
  <tabstop>replace</tabstop>
  <tabstop>replaceAll</tabstop>
 </tabstops>
 <resources>
  <include location="../Note/Note.qrc"/>
  <include location="findandreplace.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>FindAndReplace</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
